src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -269,9 +269,9 @@ int goto_instrument_parse_optionst::doit()
 
       status() << "Pointer Analysis" << eom;
       namespacet ns(goto_model.symbol_table);
-      value_set_analysist value_set_analysis(ns);
-      value_set_analysis(goto_model.goto_functions);
-      show_value_sets(get_ui(), goto_model, value_set_analysis);
+      value_set_analysist value_set_analysis;
+      value_set_analysis.analyze_all_functions(goto_model);
+      show_value_sets(get_ui(), goto_model, value_set_analysis, ns);
```

Comment:
When this is done, maybe we should move this functionality to goto-analyze as --show --value-set-analysis